graph/build.rs

```diff
@@ -1,4 +1,5 @@
 fn main() {
+    println!("cargo:rerun-if-changed=proto");
     tonic_build::configure()
         .out_dir("src/firehose")
         .format(true)
```

Comment:
This must have been why rustfmt was still being required, I inserted the `cfg!(debug_assertions)` only in the `ethereum` build script and not here. Lets add it here as well for consistency, or remove it in both places if you think that's better.

Reply:
I think requiring `rustfmt` for building is better - I already ran into the problem locally that `cargo check --release` modified the generated files because of the different format settings.